requirements.txt

```diff
@@ -23,10 +23,13 @@ flask-cors==3.0.10
 funcsigs==1.0.2
 Jinja2==3.1.2
 MarkupSafe==2.1.1
-Werkzeug==2.1.2
+Werkzeug==2.2.3
```

Comment:
Do you need this change in this PR?  We previously had this and then I reverted it because I was getting HTTP 405 responses when posting feature editing forms.

Reply:
This is not needed. I think I must've created my changes with that bad version. Will revert.